context-free.md

```diff
@@ -2,21 +2,54 @@
 
 ##  Who is this programming language for?
 
+ContextFree is for people who want to create designs with lots of similar 
+geometric images that rely on symmetry.
 
 ## What is easy to do in this language? Why is it easy?
 
+It is easy to create symmetrical geometric patterns concisely because 
```

Comment:
I'd leave out "symmetrical" since not all recursive patterns are symmetrical (in the strictest, mirror symmetry kind of way)